src/representation/dmp/src/RigidBodyDmpConfig.h

```diff
@@ -31,7 +31,6 @@ class RigidBodyDmpConfig
   void toYamlFile(std::string filepath);
   bool isValid() const;
 
-PRIVATE:
```

Comment:
If that is the only occurence of PRIVATE in this file, you can remove the macro at the top of the file.

Reply:
ok